app/actions/deployment_cancel.rb

```diff
@@ -30,9 +30,7 @@ def cancel(deployment:, user_audit_info:)
       private
 
       def valid_state?(deployment)
```

Comment:
I feel like i would just get rid of this method if we're delegating to the `DeploymentModel`

Reply:
Agreed.  Especially if the parameter here isn't being checked.

Reply:
^^